### DISCUSSION TITLE
是否能在静态html页面中直接引用element-UI plus和 vue3的cdn的js文件 来直接使用

### DISCUSSION THREAD

**Discussion by lbl1120** [element-plus/element-plus#18623] [Ideas]
是否能在静态html页面中直接引用element-UI plus和 vue3的cdn的js文件 来直接使用

  <link href="https://cdn.bootcdn.net/ajax/libs/element-plus/2.3.12/index.min.css" rel="stylesheet">
  <script src="https://cdn.bootcdn.net/ajax/libs/vue/3.3.4/vue.global.prod.js"></script>
  <script src="https://cdn.bootcdn.net/ajax/libs/element-plus/2.3.12/index.full.js"></script>

<script>
  const { createApp, ref } = Vue
  createApp({
    setup () {
      const message = ref('Hello vue!')
      return {
        message,
      }
    }
  }).use(ElementPlus).mount('#app')
</script>

**Comment by MSTY2003** [comment]
可以，我刚刚试了一下
![image](https://github.com/user-attachments/assets/54c26247-a956-47ce-8db1-5236c15817f8)
```
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Vue 3 + Element Plus Example</title>
  <link rel="stylesheet" href="https://cdn.bootcdn.net/ajax/libs/element-plus/2.3.12/index.css">
</head>
<body>
  <div id="app">
    <el-button @click="message = 'Hello Element Plus!'">{{ message }}</el-button>
  </div>

  <script src="https://cdn.bootcdn.net/ajax/libs/vue/3.3.4/vue.global.prod.js"></script>
  <script src="https://cdn.bootcdn.net/ajax/libs/element-plus/2.3.12/index.full.js"></script>
  <script>
    const { createApp, ref } = Vue;
    createApp({
      setup() {
        const message = ref('Hello Vue!');
        return { message };
      }
    }).use(ElementPlus).mount('#app');
  </script>
</body>
</html>
```
![image](https://github.com/user-attachments/assets/5d57e137-3d8c-48ea-8fd5-50c28b4fb211)
